### PR TITLE
Request for Weather-app-dev branch

### PR DESCRIPTION
**Pull request by ClaytonSiby**
This feature branch holds the main weather application, it uses the open-weather-map API to retrieve weather data for any city in the world. Users can enter a city name in the input field & click on the ( View Weather Info ) button to view the weather data related to the city they chose. The main focus of this project is to showcase the author's understanding of async JS & working with APIs. I also made use of modules for better readability & code reusability.

Additional features include:

- webpack configuration
- eslint
- stylelint

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @ClaytonSiby  👋

**You've done quite well here.**

- [ ] At present the temperature value doesn't change, it seems you are not extracting that part from the API returned value. Please check and fix.

- [ ] Also it is important that there is a change in the background image when the weather condition changes from hot to cold. Please include this feature.

- [ ] Please check other comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/modules/domBuild.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    // if (toggleTemp) {
+    //   theTemperature.innerHTML = `${TempCalculation.tempInteger(temperature)}&deg;C`;
+    //   highTemp.innerHTML = `${TempCalculation.tempInteger(highTemperature)}&deg;C`;
+    //   lowTemp.innerHTML = `${TempCalculation.tempInteger(lowTemperature)}&deg;C`;
+    // } else {
+    //   theTemperature.innerHTML = `${TempCalculation.toFahrenheit(temperature)}&deg;F`;
+    //   highTemp.innerHTML = `${TempCalculation.toFahrenheit(highTemperature)}&deg;F`;
+    //   lowTemp.innerHTML = `${TempCalculation.toFahrenheit(lowTemperature)}&deg;F`;
+    // }
+  };
```
Comment (mikenath223) [inline src/modules/domBuild.js]: - [ ] There shouldn't be commented code blocks in your project. Please remove this.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "the-weather-app",
+  "version": "1.0.0",
+  "description": "Weather application",
+  "private": "true",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```
Comment (mikenath223) [inline package.json]: - [ ] Please install the web-dev-server as running `npm start` returns an error.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @ClaytonSiby :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**